src/telegraf.ts

```diff
@@ -1,33 +1,34 @@
-/* eslint @typescript-eslint/no-var-requires: "warn" */
-import * as http from 'http'
-import * as https from 'https'
 import type * as tt from '../typings/telegram-types'
 import type ApiClient from './core/network/client'
+import BaseScene from './scenes/base'
 import Composer from './composer'
 import Context from './context'
 import crypto from 'crypto'
+import Debug from 'debug'
+import Extra from './extra'
 import generateCallback from './core/network/webhook'
+import http from 'http'
+import https from 'https'
+import Markup from './markup'
 import { promisify } from 'util'
+import Router from './router'
+import session from './session'
+import Stage from './stage'
 import Telegram from './telegram'
 import type { TlsOptions } from 'tls'
 import { URL } from 'url'
-const debug = require('debug')('telegraf:core')
-const Extra = require('./extra')
-const Markup = require('./markup')
-const session = require('./session')
-const Router = require('./router')
-const Stage = require('./stage')
-const BaseScene = require('./scenes/base')
 
-const DEFAULT_OPTIONS: Telegraf.Options<Context> = {
+const debug = Debug('telegraf:core')
+
+const DEFAULT_OPTIONS: Telegraf.Options<any> = {
   telegram: {},
   retryAfter: 1,
   handlerTimeout: 0,
   channelMode: false,
   contextType: Context,
 }
 
-const noop = () => {}
+const noop: () => void = () => undefined
```

Comment:
This is just more verbose way of doing the same thing, isn't it?

Reply:
No, this fixes a type error further down, where another function expects a void callback, and not an undefined callback

Reply:
How? `() => {}` is already a void callback.

Reply:
Weird, NOW it is for me too, previously this wasn't a void callback but the type turned into () => undefined.. uhh